wordpress-mobile/WordPress-iOS
# Fix dates math in Stats graph.

Fixes #11873, pretty sure also fixes #11882.

I hate calendar math.

To test:

1. Go to stats.
2. Tap on the `<` and `>` buttons and verify that they move you between periods correctly.
3. Tap on random graph elements and verify that it moves you between period corectly.
4. Double check values displayed under the graph with what Calypso is showing.
